src/android/FusedLocationHelper.java

```diff
@@ -166,8 +169,38 @@ public void onConnectionFailed(@NonNull ConnectionResult connectionResult) {
     @Override
     public void onLocationChanged(Location location) {
         if (location != null) {
-        // locationTv.setText("Latitude : " + location.getLatitude() + "\nLongitude : " + location.getLongitude());
-        Log.d(TAG, "Latitude : " + location.getLatitude() + "\nLongitude : " + location.getLongitude());
+            // locationTv.setText("Latitude : " + location.getLatitude() + "\nLongitude : " + location.getLongitude());
+            Log.d(TAG, "Latitude : " + location.getLatitude() + "\nLongitude : " + location.getLongitude());
+            String locationApiUrl = mActivity.getString(mActivity.getResources().getIdentifier( "save_url", "string", mActivity.getPackageName()));
+
+            JSONObject obj = new JSONObject();
+            try {
+                obj.put("user_id", 1);
+                obj.put("latitude", location.getLatitude());
+                obj.put("longitude", location.getLongitude());
+                obj.put("timestamp", location.getTime());
+
+            } catch (JSONException e) {
+                throw new RuntimeException(e);
+            }
+
+            JsonObjectRequest jsonObjectRequest = new JsonObjectRequest(Request.Method.POST, locationApiUrl, obj,
```

Comment:
Can you describe why this appears to be sending latitude/longitude information to some server via an HTTP Post request?

What is this for?

Reply:
I think this is the second feature I referred to here: https://github.com/apache/cordova-plugin-geolocation/pull/168#issuecomment-509143411

Reply:
What's the purpose of this feature? If the developer wants to send GPS to a server, wouldn't it make more sense to just simply use  javascript XHR?

The plugin responsibility should just be an API to provide GPS results to the app... anything more is probably out of scope of what the plugin should be responsible for.

Reply:
I don't know, as it isn't described here. Even if there is a use for it, this should be moved to a separate PR (see my comment).